<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssueBooks</class>
 <widget class="QDialog" name="IssueBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>386</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ======== MAIN WINDOW ======== */
QDialog {
    background-color: #f7f5d9;
    color: #3f5433;
}

/* ======== TITLE ======== */
#titleLabel {
    font-size: 32px;
    font-weight: bold;
    font-family: &quot;Comic Sans MS&quot;;
    color: #3f5433;
}

/* Кнопка &quot;Назад&quot; */
QPushButton#goBackButton {
    background-color: transparent;
    border: none;
    color: #3d4a3f;
    font-size: 20px;
    border-radius: 8px;
}

QPushButton#goBackButton:hover {
    background-color: #d7d9c6;
}

/* ======== LABELS ======== */
QLabel {
    font-size: 18px;
    color: #3f5433;
}

/* ======== LINE EDITS ======== */
QLineEdit {
    background: #ffffff;
    border: 2px solid #a6b88f;
    border-radius: 10px;
    padding: 8px 12px;
    font-size: 15px;
    color: #3f5433;
}

QLineEdit:focus {
    border: 2px solid #7f936a;
}

/* ======== BUTTONS (Issue book / Cancel) ======== */
QPushButton {
    padding: 8px 22px;
    border-radius: 10px;
    font-size: 18px;
    font-weight: bold;
}

/* ISSUE BOOK BUTTON — olive green */
#issueBookButton {
    background-color: #6c7f5f;
    color: white;
}

#issueBookButton:hover {
    background-color: #7f9270;
}

#issueBookButton:pressed {
    background-color: #5b6c50;
}

/* CANCEL BUTTON — beige/olive */
#cancelButton {
    background-color: #c7c3aa;
    color: #3f5433;
}

#cancelButton:hover {
    background-color: #d4d1bb;
}

#cancelButton:pressed {
    background-color: #b6b29b;
}

/* ======== BACK BUTTON ======== */
#backButton {
    background: transparent;
    border: none;
    color: #3f5433;
    font-size: 20px;
}
</string>
  </property>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>371</width>
     <height>371</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>350</width>
      <height>364</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
       <item>
        <widget class="QPushButton" name="goBackButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/left-arrow.png</normaloff>icons/left-arrow.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Issue books</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Student name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="studentNameLineEdit">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Student email</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="studentEmailLineEdit">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Book name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="bookNameLineEdit">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Book id</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="bookIdLineEdit">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Issue date</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="issueDateLineEdit">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="issueBookButton">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Issue book</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
